spec/features/sidebar_spec.rb

```diff
@@ -8,4 +8,22 @@
 
     expect(active_link.text).to eq "Customers"
   end
+
+  it "displays translated name of model" do
+    translations = {
+      activerecord: {
+        models: {
+          customer: {
+            one: "User",
+            other: "Users",
+          } } } }
```

Comment:
What do you feel about putting every `}` on its own line? This looks rather Lisp-y.